<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commande</class>
 <widget class="QDialog" name="commande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commande</string>
  </property>
  <widget class="QPushButton" name="pushButton_ajout_C">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tri_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
   </property>
   <property name="text">
    <string>Trier liste</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimer_C">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rechercher_C">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
   </property>
   <property name="text">
    <string>Chercher</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(233,248,255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Num de la Commande:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nom du produit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Date de l'arrivé:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>nombre de Commandes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_numC">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomP_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nbrC">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date_delait">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chercherD">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
    </property>
    <property name="text">
     <string>Chercher </string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableCommande_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>311</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Chercher une commande en saisissant le nom du produit recherché:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableCommande">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Modifier_C">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomP_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_imprimer_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
